### PR TITLE
TEST/LOG: Show logfile in case of test failure

### PR DESCRIPTION
**Pull request by yosefe**

### CODE REVIEW

**Comment by yosefe** [comment]
@dmitrygladkov can you pls take a look?

File: test/gtest/ucs/test_log.cc [hunk 1]
```diff
@@ -61,14 +63,21 @@ class log_test : public ucs::test {
         return system(cmd);
     }
 
+    std::string read_logfile() {
+        std::stringstream ss;
+        std::ifstream ifs(logfile);
```
Comment (dmitrygladkov) [inline test/gtest/ucs/test_log.cc]: does it mean that it will rewrite `logfile` name and `unlink` will unsuccessful in the dtor?

Comment (yosefe) [reply test/gtest/ucs/test_log.cc]: std::ifstream is only reading the file (before dtor)

Comment (dmitrygladkov) [reply test/gtest/ucs/test_log.cc]: > std::ifstream is only reading the file (before dtor)

I see, thx for the explanation!

**Pull request merged**